shared/js/views.jsx

```diff
@@ -831,6 +927,30 @@ loop.shared.views = (function(_, mozL10n) {
     }
   });
 
+  var RemoteCursorView = React.createClass({
+    mixins: [React.addons.PureRenderMixin],
+
+    propTypes: {
+      remoteCursorOffset: React.PropTypes.object,
+      remoteCursorPosition: React.PropTypes.object
```

Comment:
The other option is we move more of the positioning logic into `RemoteCursorView` - I'm not quite sure it makes sense, so feel free to ignore.

What I'm vaguely thinking about is you could pass it the video size and the values received for the actual remote cursor position, then have the `RemoteCursorView` do the necessary calculations for positioning it on the display.

That way, we keep the location calculations next to the item that is being positioned.